routes/auth.js

```diff
@@ -93,72 +97,32 @@ router.use('/logout', function(req, res, next) {
 });
 
 router.post('/register', validateLocalCredentials, function(req, res, next) {
-	let savedUser = null;
-	UserModel
-	.forge({email: req.body.email})
-	.fetch()
-	.then(function(u) {
-		// The account already exists, return an error.
-		if(u) {
-			res.error.add('auth.EMAIL_EXISTS', 'email');
-			res.error.send('/auth/register');
+	let userData = _.extend({}, req.body);
```

Comment:
prefer consistent use of `_.assign()` or `_.extend()`. Is there a reason to not use `Object.assign()` and thereby not require another library?